contents/docs/feature-flags/common-questions.mdx

```diff
@@ -92,6 +97,49 @@ A workaround is to create a dynamic behavioral cohort and then duplicate it as a
     classes="rounded"
 />
 
+Another option is to set a person property on the event and then target that property. For example, you can set a person property on a `signed up` event and then target the `business_industry` property with the feature flag.
+
+```js
+posthog.capture(
+  'signed up',
+  {
+    $set: { business_industry: 'finance'  },
+  }
+)
+```
+
+If you plan to call the flag immediately after the event, you will likely need to [set the property for the flag manually](/docs/libraries/js#overriding-server-properties) with `setPersonPropertiesForFlags()`.
+
+```js
+posthog.setPersonPropertiesForFlags({
+  business_industry: 'finance'
+})
+```
+
+## Why do I see multiple feature flag calls when I only expect one?
+
+This is likely because the `onFeatureFlags()` callback is invoked every time feature flags are reloaded. This happens by default whenever person properties change, you call `reloadFeatureFlags()`, or other flag reload triggers.
```

Comment:
```suggestion
This is likely because the `onFeatureFlags()` callback is invoked every time feature flags are reloaded. This happens by default when person properties change, when you call `reloadFeatureFlags()`, or when other flag-reloading triggers happen.
```

I struggled to parse the commas, but that might have made me mangle it more.